brave/brave-core
# fix(wallet): Add localization for file transaction info

Resolves https://github.com/brave/brave-browser/issues/26136

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to send page
2. Select Filecoin, enter amount, select to account and click on "Review order"
3. In the transaction confirmation screen, the transaction info labels should be the same as before.


<img width="331" alt="Screenshot 2023-01-27 at 14 10 03" src="https://user-images.githubusercontent.com/48117473/215072963-b0e9ce8e-6ed8-440f-969d-05c812d2b471.png">
